cli/dcoscli/cluster/main.py

```diff
@@ -31,8 +37,35 @@ def main(argv):
 
 @decorate_docopt_usage
 def _main(argv):
+    doc = default_doc("cluster")
```

Comment:
Can you just create an alternate text file (one that includes the {link, unlink} commands) and conditionally pull in the correct? Seems cleaner than patching the contents like this. Especially since this is a one-off hack just for this release.